internal/backend/webhookssvc/svc.go

```diff
@@ -87,12 +87,14 @@ func (s *Service) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if _, err := s.dispatcher.Dispatch(ctx, event, nil); err != nil {
+	eid, err := s.dispatcher.Dispatch(ctx, event, nil)
+	if err != nil {
 		sl.Errorw("dispatch failed", "event", event, "err", err)
 		http.Error(w, "Internal Server Error", http.StatusInternalServerError)
 		return
 	}
 
+	w.Header().Set("Autokitteh-Event-ID", eid.String())
```

Comment:
There's no rule that only the first letter in header words can be capitalized, e.g. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers#websockets

```suggestion
	w.Header().Set("AutoKitteh-Event-ID", eid.String())
```